eng/common/TestResources/New-TestResources.ps1

```diff
@@ -313,25 +311,53 @@ foreach ($templateFile in $templateFiles) {
         }
     }
 
-    foreach ($key in $deploymentOutputs.Keys)
+    if ($File)
     {
-        $value = $deploymentOutputs[$key]
-        
-        if ($CI) {
-            # Treat all ARM template output variables as secrets since "SecureString" variables do not set values.
-            # In order to mask secrets but set environment variables for any given ARM template, we set variables twice as shown below.
-            Write-Host "Setting variable '$key': ***"
-            Write-Host "##vso[task.setvariable variable=_$key;issecret=true;]$($value)"
-            Write-Host "##vso[task.setvariable variable=$key;]$($value)"
-        } else {
-            Write-Host ($shellExportFormat -f $key, $value)
+        if (!($IsWindows))
+        {
+            Write-Host "File option is supported only on Windows"
```

Comment:
Why not support mac/linux and chmod the file in ~/.azure as 600? That was my plan originally.

Reply:
Most consumers are on windows, we can add *nix later.